woodwork/table_accessor.py

```diff
@@ -825,16 +825,28 @@ def describe_dict(self, include=None, callback=None):
                 - unit (str): unit of measurement for progress/total
                 - time_elapsed (float): total time in seconds elapsed since start of call
 
+            extra_stats (bool): If True, will calculate a histogram and top values for numeric columns, top values
+                for categorical columns and recent values for datetime columns. Output can be controlled by bins,
+                top_x and recent_x parameters.
+            bins (int): Number of bins to use when calculating histogram
+                for numeric columns. Defaults to 10. Will be ignored unless extra_stats=True.
+            top_x (int): Number of values to get when getting the most frequent
+                values for categorical columns. Defaults to 10. Will be ignored unless extra_stats=True.
+            recent_x (int): Number of recent values to get when getting the most recent
+                dates to calculate frequency for datetime columns. Defaults to 10. Will be ignored unless
+                extra_stats=True.
+
         Returns:
             dict[str -> dict]: A dictionary with a key for each column in the data or for each column
             matching the logical types, semantic tags or column names specified in ``include``, paired
             with a value containing a dictionary containing relevant statistics for that column.
         """
         if self._schema is None:
             _raise_init_error()
-        return _get_describe_dict(self._dataframe, include=include, callback=callback)
+        return _get_describe_dict(self._dataframe, include=include, callback=callback,
+                                  extra_stats=extra_stats, bins=bins, top_x=top_x, recent_x=recent_x)
 
-    def describe(self, include=None, callback=None):
+    def describe(self, include=None, callback=None, extra_stats=False, bins=10, top_x=10, recent_x=10):
```

Comment:
The extra stats added here aren't really understandable from the dataframe format. 

![image](https://user-images.githubusercontent.com/64278226/122241150-53b64a00-ce90-11eb-925a-cbea2b094a9b.png)

I guess maybe their purpose isn't to be useful for a quick onceover the way the other stats are in this format, so maybe that's okay. Just wanted to point it out, though.

Reply:
Yah these stats don't fit in neatly in a DataFrame, which is why they are off by default with doing `df.ww.describe`

Reply:
We could leave this option out of `describe` and only include it for `describe_dict`?

Reply:
Will only use this in `describe_dict` for now.